x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_grid_page.tsx

```diff
@@ -167,7 +165,7 @@ export class APIKeysGridPage extends Component<Props, State> {
 
     if (!isLoadingTable && apiKeys && apiKeys.length === 0) {
       return (
-        <EuiPageContent>
+        <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
```

Comment:
Should the create button in the empty prompt use the same "iconInCircleFilled" icon as the ones in the header on the index page? Seems a bit inconsistent otherwise.

Reply:
Ah I pushed a commit for this before I realized you asked @cchaos for input. I updated this to be `plusInCircleFilled` as that's consistent with the other "Create" buttons, but I'm happy to adjust as necessary

Reply:
For `fill` style buttons, the `Filled` icon is best :) sorry for the late response

Reply:
@cchaos Sorry, I was talking about the empty prompts. 

We show an icon in the CTA on the index pages:

<img width="1249" alt="Screenshot 2021-06-24 at 09 28 15" src="https://user-images.githubusercontent.com/190132/123229871-9ac6bf80-d4ce-11eb-8e0e-983df9ac0d54.png">

But we do not do that for the empty state:

<img width="619" alt="Screenshot 2021-06-24 at 09 27 28" src="https://user-images.githubusercontent.com/190132/123229910-a619eb00-d4ce-11eb-90ef-4d3aa6d3ad7f.png">

Seems inconsistent and I was wondering if that's intentional.

Reply:
THere's no real pattern for the empty state version of CTA's. You can add one if you'd like. I wouldn't hold up this PR for it though 😉